wgpu/backends/rs_mappings.py

```diff
@@ -236,23 +251,25 @@
 enum_str2int = {
     "BackendType": {
         "Null": 0,
-        "D3D11": 1,
-        "D3D12": 2,
-        "Metal": 3,
-        "Vulkan": 4,
-        "OpenGL": 5,
-        "OpenGLES": 6,
+        "WebGPU": 1,
```

Comment:
Huh? How can webgpu be a webgpu backend?

Reply:
Good question. On WebGPU you can't select the backend, so its not that it leaked from that spec. Odd...

Reply:
The value is the same as D3D11, which is also suspect. 🤔

Reply:
https://github.com/gfx-rs/wgpu/blob/76bcd9c494a4359853bc2caec8eebbaa990a34e3/wgpu-info/src/main.rs#L127 webgpu backend is used to indicate browser apparently

Reply:
Ah, int value should be 6: https://github.com/gfx-rs/wgpu/blob/1ee6036f440210e038098bcc372ddd738437918f/wgpu-types/src/lib.rs#L64

Reply:
OpenGLES seems to have been replaced?

Reply:
> webgpu backend is used to indicate browser apparently

Right, maybe just to signal to wgpu-core that its running in a browser, maybe having some extra checks or constraints.

I can imagine that OpenGLES is dropped because it does not have compute shaders for instance. Not sure why it has ever been a backend.